### ISSUE TITLE
shaka777

### ISSUE THREAD

**Issue by shaka777** [bitpay/copay#6120]
Guys I need help!
I have a Copay wallet and attempted to transfer a small amount of funds.
The problem is, I forgot to insert the receiving wallet code.
How do I correct this situation?

**Comment by my100bucks** [comment]
Did you left it empty? Or you put the wrong code?

**Comment by shaka777** [comment]
I left it empty!

**Comment by shaka777** [comment]
Sorry....I did not mean to close this issue.

**Comment by shaka777** [comment]
I did not insert the recipient bitcoin address. I left it blank and sent a small amount of funds.
How can I retrieve these funds or add an address to get the funds to the recipient?

**Comment by JDonadio** [comment]
Sorry but I don't understand the situation here, can you please tell me what version are you running and also how did you send funds without a recipient address? I'm trying to reproduce it...

**Comment by shaka777** [comment]
Hi Javier,

I'm using version - v3.2.1

I intended to send 0.014 btc to a recipient. I forgot to insert the
recipient address in the address bar ( I left it empty ). I then hit the
send button. It looks as though the transaction is going through the
blockchain without an address???

Can I still attach an address to this transaction at this stage? or can I
cancel this transaction at this stage?
Not sure what to do!

Mike

**Comment by shaka777** [comment]
Also Javier,

Normally once funds are sent and the transaction is complete it says "sent
funds"
The message I have is for the transaction I'm having a problem with is:
"moved funds".

**Comment by mnfirdaus** [comment]
try again...

**Comment by gabrielbazan7** [comment]
@shaka777 you send your funds to the same wallet, that's why it shows moved

**Comment by shaka777** [comment]
I've attempted to send the funds again as a new transaction.

I pasted the bitcoin address...then the app takes me straight to confirm
with no option to enter an amount. the amount selected is 0.001 btc which
is the same amount as first successful transaction to the same address. The
second transaction (the problem transaction) for a different amount has
been "moved" and not sent.

**Comment by gabrielbazan7** [comment]
Could you paste the bitcoin address here ?? 

In the recipient input you have to paste the address of the wallet where you want to send the transaction, then you enter the amount you want to send and then on the confirm view you choose the sender wallet.

**Comment by shaka777** [comment]
Gabriel...I've managed to unlock the funds...for some reason, Copay would
not allow me to send the funds to the same address as the first
transaction. What I did was sent the funds to another wallet of mine with a
different address. That worked. Yes I did have to pay fees twice. Once with
the problem transaction and again sending it to my other wallet.

Thanks all the same to all of you trying to help me! Much appreciated.

Mike
